### ISSUE TITLE
Do not copy date of report from travel entry to resulting case

### ISSUE THREAD

**Issue by bernardsilenou** [hzi-braunschweig/SORMAS-Project#6493]
### Feature Description

When creating a case from travel entry, the report date of travel entry is used as the report date of the case. 
### Proposed Change

- When creating a case from travel entry, keep report date as null and ask user to specify a date before saving.

**Comment by bernardsilenou** [comment]
@kwa20 Please confirm is its oK

**Comment by kwa20** [comment]
@bernardsilenou agreed, reason being that if the case is created later, the date of report is not necessarily equal to the date of report of the travel entry.
